plugins/deployment_manager/DeploymentManager.ts

```diff
@@ -133,46 +146,105 @@ export class DeploymentManager {
     );
   }
 
+  /* Unconditionally casts a contract as the given artifact type, without caching */
+  async cast<C extends Contract>(address: string, artifact: string): Promise<C> {
```

Comment:
By no means suggesting now, but we should probably break these functions out into modules with tests at a later point when we have time.

Reply:
Yeah, although by nature if you don't get the type of contract you want, you just won't be able to make your calls, so they are kind of tested by all the tests already (e.g. `cast` is core part of `deployComet` now to get the handle before the impl is set, which is why this was added)